stack.py

```diff
@@ -204,6 +207,15 @@ def parse_args():
         help="XLA commit reference to checkout on clone",
         default=XLA_REPO_REF,
     )
+    dev.add_argument(
+        "--xla-dir",
+        help=(
+            "Set the XLA path in the Makefile. Note, that this script will "
+            "still clone XLA into ./xla, it just won't be used in the "
+            "Makefile."
```

Comment:
what if I don't want to git clone another XLA to `./xla` ? can I directly just use the local existing xla repo?